dts/arm/renesas/smartbond/da1469x.dtsi

```diff
@@ -199,6 +199,24 @@
 			status = "disabled";
 		};
 
+		uart2: uart@50020100 {
```

Comment:
For this commit, could you amend the commit summary to include "smartbond" somewhere? For example, `dts: serial/smartbond: add uart2 and uart3 definitions`. Current summary will confuse people looking at the git log as it says adding uart2 and uart3 for all.

Reply:
fixed